packages/botonic-react/src/components/multichannel/multichannel-text.jsx

```diff
@@ -287,12 +287,15 @@ export const MultichannelText = props => {
     return (
       <>
         {texts &&
-          texts.map((e, i) => (
+          texts.map((message, i) => (
```

Comment:
if you are not using the first argument, just do `texts.map(_, i) => ...`. it was confusing to me when reviewing